packages/typescript/rollup.config.js

```diff
@@ -1,21 +1,15 @@
-import buble from '@rollup/plugin-buble';
+import typescript from '@rollup/plugin-typescript';
```

Comment:
Shouldn't this be a relative import from local code?

Reply:
Making it a relative import depends on the code already being compiled in order to compile it. We kind of follow this pattern already as we don't depend on relative packages in the monorepo, instead depending on the npm version of pluginutils, babel, etc.

Reply:
Thanks, I must have never noticed! 👍